common/addressing/defaults.go

```diff
@@ -32,7 +32,7 @@ const (
 	// Default IPv4 prefix length of entire cluster
 	DefaultIPv4ClusterPrefixLen = 8
 	// Default IPv6 prefix to represent NATed IPv4 addresses
-	DefaultNAT46Prefix = "aa46::/48"
+	DefaultNAT46Prefix = "f00d::/48"
```

Comment:
yeah this is the mapping ::
f00d:0000:0000:0000:<node>:<node>:<state>:<lxc>/128
f00d:0000:0000:0000:<node>:<node>:<ipv4address>/128

and the ipv4 address pretty much overlaps with the <state>:<lxc> completely. If we want the nat46 functionality to work so containers are reachable from the host, simply using the HOST_IP as prefix for both SRC and DST NAT prefix will solve the issue. The only pending problem there is making sure the ipv4 address does not conflict with an existing ipv6 address. I need to try it out with a conflict but even if it conflicts since we do the conntrack lookup first we will convert it to v6 before sending the packet out. But I agree its best if there is no conflict. 

However if we want a container to be able to reach another container using nat46 I think it needs more datapath changes. Is that something you think is also useful ?

Thanks.